src/lib/api/fareharbour/models/FareHarbourItem.ts

```diff
@@ -0,0 +1,84 @@
+import { MaybeString } from "@/lib/common";
+
+export type FareHarbourItemsResult = {
+    items: FareHarbourItem[];
+}
+
+
+type FareHarbourItem = {
+    customer_prototypes : CustomerPrototype[];
+    description: string;
+    desription_bullets: string[] | null | undefined;
+    description_safe_html: string;
+    description_text: string;
+    headline: string;
+    images: FareHarbourImage[];
+    name: string;
+    pk: number;
+    primary_location: FareHarbourLocation;
+    structured_description: FareHarbourStructuredDescription;
+    health_and_safety_policy: MaybeString;
+    health_and_safety_policy_safe_html: MaybeString;
+}
+
+type FareHarbourStructuredDescription = {
+    accessibility: MaybeString;
+    cancellation_summary: MaybeString;
+    check_in_details: MaybeString;
+    description: MaybeString;
+    disclaimers : MaybeString;
+    duration: MaybeString;
+    extras: MaybeString;
+    faqs: MaybeString;
+    group_size: MaybeString;
+    highlights: MaybeString;
+    itinerary: MaybeString;
+    max_age: number | null | undefined;
```

Comment:
All of these could be set as optional properties like `group_size?: string;`. What are your thoughts on this?